src/components/List.js

```diff
@@ -0,0 +1,14 @@
+//import useLocalStorageState from "use-local-storage-state"
+
+export function List({activities}) {
+    const listItems = activities.map((activity) => {
+        return  <li key={activity.id} 
+        goodweather={activity.goodweather}
+        >{activity.name} - is good for outdoor: {activity.goodweather ? "Yes" : "No"}</li>
+    })
+  
+return (
+        <div>
+            <ul>{listItems}</ul>
+        </div>
+)}
```

Comment:
Another way could be to map directly inside the return:

```suggestion
export function List({ activities }) {
  return (
    <div>
      <ul>
        {activities.map((activity) => {
          return (
            <li key={activity.id} goodweather={activity.goodweather}>
              {activity.name} - is good for outdoor:{" "}
              {activity.goodweather ? "Yes" : "No"}
            </li>
          );
        })}
      </ul>
    </div>
  );
}
```

Reply:
thx